recipe/meta.yaml

```
@@ -36,11 +40,15 @@ test:
 
 about:
   home: https://pagure.io/python-daemon/
+  description: |
+    This library implements the well-behaved daemon specification of
+    :pep:`3143`, “Standard daemon process library”.
   license: Apache-2.0
   license_family: APACHE
   license_file: LICENSE.ASF-2
```

```suggestion
  license: Apache-2.0 AND GPL-3.0-or-later
  license_family: Other
  license_file: 
    - LICENSE.ASF-2
    - LICENSE.GPL-3
```
The readme mentions both license being required.

---

It is technically Apache-2.0 only if we only distribute the library without anything else.